packages/scene-composer/src/components/SceneComposerInternal.spec.tsx

```diff
@@ -0,0 +1,220 @@
+import * as React from 'react';
```

Comment:
overall these tests heavily rely on snapshots, which are not very good tests to give people certainty that changes are correct, as they have a tendancy to fail due to _any_ changes, and are often blindly updated.

Please instead assert on the expected output, hopefully in a way that they only break if the thing the test is trying to assert on, is actually wrong

Reply:
We've been trying to mock the child components that will change for unrelated changed, and they are working fine so far. so we can keep them like this for now and switch to assert on the expected output once we see issues